src/main/webapp/js/reducers/index.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+import vaccineReducer from './vaccinee-reducer';
+
+const rootReducer = combineReducers({
+  vaccineReducer,
```

Comment:
I think the state should be called vaccinee, so change it to `vaccinee: vaccineReducer`